packages/mco/components/modals/app-failover-relocate/subscriptions/peer-cluster-status.tsx

```diff
@@ -156,14 +156,29 @@ export const PeerClusterStatus: React.FC<PeerClusterStatusProps> = ({
   const [peerStatus, setPeerStatus] = React.useState<StatusType>(
     initalPeerStatus(t)
   );
+
   const setErrorMessage = React.useCallback(
-    (errorMessage: ErrorMessageType) => {
+    (errorMessage: ErrorMessageType | NoError) => {
+      if (errorMessage === 0) {
```

Comment:
```suggestion
      if (errorMessage === null) {. (or) if (errorMessage ===0 as ErrorMessageType) 
```
call setErrorMessage(null) or setErrorMessage(0 as ErrorMessageType) for resetting

Reply:
Ack

Reply:
thank you, i will go with errorMessage ===0 as ErrorMessageType as we already doing it in other places